device/mellanox/x86_64-mlnx_msn3700-r0/ACS-MSN3700/buffers.json.j2

```diff
@@ -0,0 +1 @@
+../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers.json.j2
```

Comment:
not sure i understand what you did
you created a new file for buffers for sn3700 but you are using sn2700 buffers
according to the excel the buffer for sn3700 are not the same for sn2700

Reply:
```suggestion
../../x86_64-mlnx_msn2700-r0/ACS-MSN2700/buffers.json.j2
```
This is a template which are common for all our platforms.
It does not contain any config itself:
{%- set default_topo = 't0' %}
{%- include 'buffers_config.j2' %}